udata_hydra/cli.py

```diff
@@ -124,6 +122,23 @@ async def load_catalog(url=CATALOG_URL):
         os.unlink(fd.name)
 
 
+@cli
+async def check_url(url, method="get"):
+    """Quickly check an URL"""
+    log.info(f"Checking url {url}")
+    async with aiohttp.ClientSession(timeout=None) as session:
```

Comment:
Wouldn't we want to use the check_url core function? (We would need to factorize it)

Reply:
Don't worry, I've another helper coming for that ;-)